packages/react-scripts/config/zion-webpack.config.js

```diff
@@ -0,0 +1,50 @@
+require('dotenv').config()
+
+const initiatedDirectory = process.env.INIT_CWD
+const nodeExternals = require('webpack-node-externals')
+const merge = require('webpack-merge')
+const setupProxy = require('../proxy/setupProxy')
+const baseWebpack = require('./webpack.config.js')
+
+const craWebpack = baseWebpack('development')
+delete craWebpack.entry
+delete craWebpack.output
+delete craWebpack.plugins
+delete craWebpack.optimization
+
+module.exports = merge(craWebpack, {
+  externals: [nodeExternals(), 'react', 'react-dom', /^@fs/, /^@fs-sandbox/],
```

Comment:
didn't we have prop-types in the explicit externals array here? and @emotion/core?

Reply:
I don't know. I copied this from https://github.com/fs-webdev/configurations/blob/master/webpack.config.js#L16

Reply:
oh yeah. I added those 2 specifically inside of zion.